custom-elements/reactions/HTMLSourceElement.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
```

Comment:
This file introduces new helpers which require a lot more work to review. Could you do that in a separate PR that I, or someone else, can dedicate more time to?

Reply:
Sure，I‘ve done it in pr https://github.com/web-platform-tests/wpt/pull/16696